packages/docusaurus/src/commands/swizzle.ts

```diff
@@ -149,7 +149,8 @@ export default async function swizzle(
 ): Promise<void> {
   const context = await loadContext(siteDir);
   const pluginConfigs = loadPluginConfigs(context);
-  const pluginNames = getPluginNames(pluginConfigs);
+  const flatPluginConfigs = [...pluginConfigs.plugin, ...pluginConfigs.theme];
```

Comment:
To be nitpicky: Don't use `flatPluginConfigs`, IMO it's a bad name, because it implies implementation details. Maybe `const pluginConfigs = [...rawPluginConfigs.plugin, ...rawPluginConfigs.theme]`?